src/Uid2Options.ts

```diff
@@ -6,7 +6,7 @@ import { InitCallbackOptions } from "./Uid2InitCallbacks";
 export type Uid2Options = BaseUid2Options &
   InitCallbackOptions &
   UID2CookieOptions &
-  Uid2ApiClientOptions;
+  Uid2ApiClientOptions & { useCookie?: boolean; };
```

Comment:
I think you could add useCookie to the base options type - I think that's just there for options that are just for `uid2sdk.ts` and not provided by one of the other classes/files.

Reply:
done!